Description: Fix failures build on m68k, mips and hppa.
Author: Mateusz Łukasik <mati75@linuxmint.pl>

--- a/src/inotify/inotify-syscalls.h
+++ b/src/inotify/inotify-syscalls.h
@@ -39,6 +39,32 @@
 # define __NR_inotify_init	290
 # define __NR_inotify_add_watch	291
 # define __NR_inotify_rm_watch	292
+#elif defined (__hppa__)
+# define __NR_inotify_init      269
+# define __NR_inotify_add_watch 270
+# define __NR_inotify_rm_watch  271
+#elif defined (__mips__)
+# include <sgidefs.h>
+# if _MIPS_SIM == _MIPS_SIM_ABI32
+#  define __NR_Linux             4000
+#  define __NR_inotify_init      (__NR_Linux  284)
+#  define __NR_inotify_add_watch (__NR_Linux  285)
+#  define __NR_inotify_rm_watch  (__NR_Linux  286)
+# elif _MIPS_SIM == _MIPS_SIM_ABI64
+#  define __NR_Linux             5000
+#  define __NR_inotify_init      (__NR_Linux  243)
+#  define __NR_inotify_add_watch (__NR_Linux  244)
+#  define __NR_inotify_rm_watch  (__NR_Linux  245)
+# elif _MIPS_SIM == _MIPS_SIM_NABI32
+#  define __NR_Linux             6000
+#  define __NR_inotify_init      (__NR_Linux  247)
+#  define __NR_inotify_add_watch (__NR_Linux  248)
+#  define __NR_inotify_rm_watch  (__NR_Linux  249)
+# endif
+#elif defined (__mc68000__)
+# define __NR_inotify_init       284
+# define __NR_inotify_add_watch  285
+# define __NR_inotify_rm_watch   286
 #else
 # error "Unsupported architecture!"
 #endif
